client/Cargo.toml

```diff
@@ -31,6 +31,7 @@ tower = "0.4"
 tracing = "0.1"
 url = "2.2"
 uuid = { version = "1.1", features = ["v4"] }
+lazy_static = "1.4.0"
```

Comment:
Drive-by comment: it is my understanding that lazy-static is more or less deprecated, and people should be using once_cell instead -- which offers a more idiomatic anyway. In newer versions, you can even rely on `OnceLock` from `std` directly, although that is a little less ergonomic than once_cell's `Lazy`.

Reply:
https://github.com/temporalio/sdk-core/pull/694